README.md

```diff
@@ -46,7 +40,7 @@ This project comes with default VM SKU test configuration files that list the te
 ## Usage ##
 
 - Invoke health checks using a script that determines SKU and runs the configuration file according to SKU for you:
-```sudo ./run-health-checks.sh [-h|--help] [-c|--config <path to an NHC .conf file>] [-o|--output <directory path to output all log files>] [-a|--all_tests] [-v|--verbose]```
+```sudo ./run-health-checks.sh [-h|--help] [-c|--config <path to an NHC .conf file>] [-o|--output <directory path to output all log files>] [-e|--append_conf < path to conf file to be appended >] [-a|--all_tests] [-v|--verbose]```
```

Comment:
Is there any special required format for the conf file to be appended? Maybe you could add an example?

Reply:
Good point. I will address